src/api/roles/coach/coach.py

```
@@ -65,9 +66,83 @@ def create_coach_request(coach_details: CoachRequestInput, db = Depends(get_sess
+
 @router.post("/me", response_model=CoachAccountResponse)
 def me(db = Depends(get_session), acc: Account = Depends(get_coach_account)):
     return CoachAccountResponse(
         base_account=acc,
         coach_account=db.get(Coach, acc.coach_id)
-    )
\ No newline at end of file
+    )
+
+@router.post("/accept_client_request", response_model=AcceptedClientResponse)
+def accept_client_request(request_input : ClientCoachRequestInput, db = Depends(get_session), acc: Account = Depends(get_client_account)):
```

Ruff (0.15.9)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)